git_machete/cmd.py

```diff
@@ -399,7 +395,7 @@ def read_definition_file(self, verify_branches: bool = True) -> None:
                 self.annotations[b] = b_a[1]
             if b in self.managed_branches:
                 raise MacheteException(
-                    f"{MacheteClient.definition_file_path}, line {idx + 1}: branch `{b}` re-appears in the tree definition. {hint}")
+                    f"{self.definition_file_path}, line {idx + 1}: branch `{b}` re-appears in the tree definition. {hint}")
```

Comment:
While you're here... pls expand `idx` into `index`, `pfx` into `prefix` whereever you find one